src/main/java/org/maxgamer/quickshop/command/subcommand/SubCommand_Create.java

```diff
@@ -127,7 +127,7 @@ public void onCommand(
             if (p.isOnline()) {
                 Result result = plugin.getPermissionChecker().canBuild(p, b);
                 if (!result.isSuccess()) {
-                    MsgUtil.sendMessage(p, MsgUtil.getMessage("3rd-plugin-build-check-failed", p, result.getMessage()));
+                    MsgUtil.sendMessage(p, MsgUtil.getMessage("->3rd-plugin-build-check-failed", p, result.getMessage()));
```

Comment:
This is a string key and points to a language file string, you shouldn't touch it.